### PR TITLE
Hotfixes/homepage

### PR DESCRIPTION
**Pull request by yowakita**
Lots of small/medium sized tickets addressed here, the largest one being that I removed bootstrap buttons and replaced them with buttons from the Materialize library.

- [x] [Clean up homepage](https://www.pivotaltracker.com/story/show/122205803)
- [x] [Remove period from homepage hero tagline](https://www.pivotaltracker.com/story/show/122205733)
- [x] [Remove drop shadow from hero logo](https://www.pivotaltracker.com/story/show/122205409)
- [x] [Make hero logo border thicker (6px)](https://www.pivotaltracker.com/story/show/122205413)
- [x] [replace header background image file (optimized)](https://www.pivotaltracker.com/story/show/122171581)
- [x] [adjust buttons interaction](https://www.pivotaltracker.com/story/show/122205449)
- [x] [Header bg image is loading very slowly - ensure it's caching](https://www.pivotaltracker.com/story/show/122205719)
- [x] [Down arrows should jump to anchor for next section](https://www.pivotaltracker.com/story/show/122205419)
- [ ] [clean up buttons ](https://www.pivotaltracker.com/story/show/122205433)
- [x] [ensure footer links are going to the correct pages](https://www.pivotaltracker.com/story/show/122139719)
- [x] [when a user is reading a blog post, the "Blog" item in the nav should be activated (underlined) ](https://www.pivotaltracker.com/story/show/122140085)

### CODE REVIEW

File: _sass/materialize/components/_buttons.scss [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  vertical-align: middle;
+  // Gets rid of tap active state
+  -webkit-tap-highlight-color: transparent;
+}
+
+// Disabled shared style
+.btn.disabled,
+.btn-floating.disabled,
+.btn-large.disabled,
+.btn:disabled
```
Comment (ianflorentino) [inline _sass/materialize/components/_buttons.scss]: @yowakita I think you're missing a comma here

Comment (yowakita) [inline _sass/materialize/components/_buttons.scss]: @ianflorentino hm weird i just copied this whole from from a zipped up materialize folder without making any changes

Comment (yowakita) [inline _sass/materialize/components/_buttons.scss]: I think the rule is saying apply the disabled class to btn.btn-large and btn.btn-floating. Doesn' t seem to cause any compiling error and we don't use the disabled pseudo at the moment.

Comment (ianflorentino) [inline _sass/materialize/components/_buttons.scss]: @yowakita ahh yea the newline threw me off. It's suppose to be `btn.disable btn.-large:disabled`

**Comment by ianflorentino** [comment]
@yowakita which buttons were supposed to be cleaned up? I don't see any o those changes on the homepage?

**Pull request merged**